docs/DeveloperGuide.md

```diff
@@ -29,6 +29,30 @@ a Command Line Interface (CLI).
 
 
 ### 1.5 Overview of Application
+In order to better understand how DonnaFin helps financial advisors, it is useful to understand and model the status
+quo without DonnaFin.
+
+Below is an Object-Oriented Domain Model (OODM) modelling the existing state of affairs between a Financial Advisor and
+their Client/s.
```

Comment:
```suggestion
Below is an Object-Oriented Domain Model (OODM) modelling the existing state of affairs between a financial advisor and
their client/s.
```
This would be a noun, not a proper noun or a class name.